README.adoc

```diff
@@ -0,0 +1,8 @@
+= toolchain-operator
+
+This is the CodeReady Toolchain Operator repository
```

Comment:
maybe we should change this to member operator

Reply:
did you mean `host operator`?

Reply:
yes

Reply:
@dipak-pawar see the change in https://github.com/codeready-toolchain/host-operator/pull/2/commits/512ab01502cb0b66115ce341552fd0dc873cf283